src/package_io/parse_input.star

```diff
@@ -382,15 +382,15 @@ def default_participant():
         "v_max_mem": 0,
         "validator_count": None,
         "snooper_enabled": False,
-        "count": 1,
+        "count": 2,
     }
 
 
 def get_default_mev_params():
     return {
         "mev_relay_image": MEV_BOOST_RELAY_DEFAULT_IMAGE,
         # TODO replace with flashbots/builder when they publish an arm64 image as mentioned in flashbots/builder#105
-        "mev_builder_image": "ethpandaops/flashbots-builder:main",
+        "mev_builder_image": "flashbots/builder:1.13.2.4844.dev5-6e6c723",
```

Comment:
should we use a dev version?

Reply:
They don't have a recent enough version which supports pebble. So we kinda need to.